apps/hash-graph/lib/graph/src/ontology/entity_type.rs

```diff
@@ -407,14 +413,17 @@ impl fmt::Display for EntityTypeQueryPath<'_> {
             Self::PropertyTypeEdge {
                 edge_kind: OntologyEdgeKind::ConstrainsPropertiesOn,
                 path,
+                ..
```

Comment:
I would prefer a wildcard here instead (so `inheritance_depth: _`) while virtually the same thing, doing this instead of the rest pattern means that this will error out when a new field is added aiding in Compiler Driven Development ™️

Reply:
Typically I would agree but the edges on the query paths are not really well designed in the code. We have far too many wildcards around these variants, the `..` is not really a deal-breaker. My goal is to remove the generic edge variants with the actual edge names so it's possible to maybe enable `wildcard_enum_match_arm`. Sadly, there is not a lint such as `match_field_pattern` lint.
Adding all variants to the edges would be pretty verbose as some edges have up to four parameters and only adding the wildcard in a few locations seems inconsistent.